File: Niveau 1/5 – Tests et conditions/7 - Concours de tir à la corde.py
nbMembre = int(input())
poidsE1 = 0
poidsE2 = 0
for i in range(nbMembre*2):
   if i % 2 == 0:
      poidsE1 += int(input())
   else :
      poidsE2 += int(input())
if poidsE1 > poidsE2:
   print("L'équipe 1 a un avantage")
else :
   print("L'équipe 2 a un avantage")
print("Poids total pour l'équipe 1 :", poidsE1) 
print("Poids total pour l'équipe 2 :", poidsE2)   

File: Niveau 1/5 – Tests et conditions/7 - Concours de tir à la corde.cpp
#include <iostream>
using namespace std;
int main()
{
  int nbParTeam, t1 = 0, t2 = 0;
  cin >> nbParTeam;
  for (int i = 0; i < nbParTeam; i++)
  {
    int _t1, _t2;
    cin >> _t1 >> _t2;
    t1 += _t1;
    t2 += _t2;
  }
  cout << "L'équipe " << (t1 > t2 ? 1 : 2) << " a un avantage" << endl;
  cout << "Poids total pour l'équipe 1 : " << t1 << endl;
  cout << "Poids total pour l'équipe 2 : " << t2 << endl;
}